packages/uikit-workshop/package.json

```diff
@@ -93,15 +89,13 @@
     "iframe-resizer": "^3.6.2",
     "jquery": "^3.3.1",
     "js-cookie": "^2.2.0",
-    "jshint": "^2.9.5",
     "mousetrap": "^1.6.2",
     "preact-compat": "^3.18.4",
     "preact": "^8.3.1",
     "prism": "^4.1.2",
     "prismjs": "^1.15.0",
     "pwa-helpers": "^0.8.3",
     "redux-thunk": "^2.3.0",
```

Comment:
Keep in mind that we’ll need this `react-animate-height` package once #960 hits so we’ll need to be adding this right back in... thoughts?

Reply:
at first this was super odd to me that it was removed, because I did not recall doing it (it was late though 🕦)

this package is already in the same list: [here](https://github.com/pattern-lab/patternlab-node/pull/966/files#diff-e756faf6983689c170147ebe05d614d4L81) and I suspect npm removed it automatically.

I've seen some tools that alphabetize dependencies. but 🤷‍♂️ 🕵️